test/quantization/eager/test_quantize_eager_qat.py

```diff
@@ -911,6 +913,16 @@ def test_qat_embedding_bag_errors(self):
                                     "torch.per_channel_affine_float_qparams"):
             nnqat.EmbeddingBag.from_float(embed_bag)
 
+    def test_embedding_qat_qconfig_equal(self):
+        # Embedding QAT uses a NoopObserver class for activation,
+        # and a FakeQuant for weight, make sure that qconfig comparison
+        # functions properly for a mix of partial function and class in
+        # qconfig.
+        model = ManualEmbeddingBagLinear().train()
+        model = prepare_qat(model)
+
+        self.assertTrue(qconfig_equals(model.emb.qconfig,
```

Comment:
does the reverse fail?

self.assertTrue(qconfig_equals(default_embedding_qat_qconfig,model.emb.qconfig))?

Reply:
nevermind I see what is going on